ui/src/app/pages/admin/admin-user-profile.tsx

```diff
@@ -210,6 +225,20 @@ const EditableFields = ({
             highlightOnChange
             onChange={(event) => onChangeInstitution(event.value)}
           />
+          {showGoToInstitutionLink && (
+            <a
+              style={{ paddingTop: '2.3rem', paddingLeft: '0.6rem' }}
+              href={`admin/institution/edit/${updatedProfile.verifiedInstitutionalAffiliation?.institutionShortName}`}
+              target='_blank'
+            >
+              <TooltipTrigger
+                content={`Click here to go to 
```

Comment:
we should add the word "the" here: 

Click here to go to the 'Broad Institute' Details Page